src/Audit/AppInfo.php

```diff
@@ -18,7 +18,7 @@ public function audit(Sandbox $sandbox) {
     $client = $this->container->get('acquia.cloud.api')->getClient();
     $app = $this->target['acquia.cloud.application']->export();
     $this->set('drush', $this->target['drush']->export());
-
+    $this->set('environment', $this->target['acquia.cloud.environment']->export());
```

Comment:
The target is already available in policy expression to be evaluated so is this really required?